src/main/java/net/snowflake/ingest/streaming/internal/ChannelCache.java

```diff
@@ -43,6 +56,46 @@ void addChannel(SnowflakeStreamingIngestChannelInternal<T> channel) {
     }
   }
 
+  /**
+   * Get the last flush time for a table
+   *
+   * @param fullyQualifiedTableName fully qualified table name
+   * @return last flush time in milliseconds
+   */
+  Long getLastFlushTime(String fullyQualifiedTableName) {
+    return this.lastFlushTime.get(fullyQualifiedTableName);
+  }
+
+  /**
+   * Set the last flush time for a table as the current time
+   *
+   * @param fullyQualifiedTableName fully qualified table name
+   * @param lastFlushTime last flush time in milliseconds
+   */
+  void setLastFlushTime(String fullyQualifiedTableName, Long lastFlushTime) {
+    this.lastFlushTime.put(fullyQualifiedTableName, lastFlushTime);
```

Comment:
I see jitter when adding to the map in addChannel, but nothing here. 
1. Is there any other place where jitter was added?
2. Can you call setLastFlushTime() from addChannel() so there's only one place where lastFlushTIme.put happens?

Reply:
Added.